File: test/sort_test.exs
defmodule SortTest do
  use ExUnit.Case
  doctest Sort

  test "Sort.get/0 returns the list of sort records" do
    list = Sort.get() |> dbg
    # sample sort we know is in the list
    sample = %Sort{
      code: "DESC",
      id: 1,
      label: "Newest first"
    }

    assert Enum.member?(list, sample)
  end
end

File: test/sort_test.dart
import 'package:srt/srt.dart';
import 'package:test/test.dart';

void main() {
  test('Fetching sort', () async {
    final sortList = await Sort.get();
    expect(sortList.length, greaterThan(0));
    expect(sortList.first.id, 1);
  });
}
